Chapter19/19-4/otp_testing_tests.erl

```diff
@@ -0,0 +1,69 @@
+-module(otp_testing_tests).
+-import(otp_testing, [start_link/1, init/1, set_status/2, delete_disabled/0,
+                      handle_call/3, stop/0, handle_cast/2, terminate/2,
+                      add_usr/2]).
+
+% -define(NOTEST, true).
+-include_lib("eunit/include/eunit.hrl").
+
+path() -> "/Users/daniel.harris/Documents/GitErlang/learning-erlang/Chapter19/19-4/DB".
+
+
+%% Note -----------------------------------------------------------------
+% For some reason I can only test one thing at a time, otherwise they
```

Comment:
make sure that processes that you spawned are stopped after each test. Or create a unit test, that includes all tests as a sequential list of commands, for instance: start_link, add_user, set_service, lookup_id, set_status, service_flag and then stop. All of those actions ideally should be tested.

Reply:
Code now starts the server (as a test) runs other tests and stops the server (as a test) all in one go now, so nothing overlaps